### ISSUE TITLE
Usability: profile page introduction has max character limit but no helping message

### ISSUE THREAD

**Issue by fedpettinella** [loomio/loomio#4742]
I just tried to write an introduction on my profile, but it was too long. There's a validation rule in place saying it should be less than 500 characters. That's fine, but I don't know how many characters I've written.

I'd like for there to be a character count when writing in the profile box that says something like "x/500 characters". This could be there all the time, or only when I've exceeded 80% of the character limit. 

I'm new to the loomio repository, and new to the Ruby language, so I wan't able to find a fix that would do this in the time that I have. However, I do think that the fix has to do with the interaction of one or more of the files that I'll list below. I wasn't able to find the location of the actual validation rules, though. If someone could point me to that gap in my knowledge then I could work on a pull request when I have time.

profile page: https://github.com/loomio/loomio/tree/master/client/angular/pages/profile_page

form helper: https://github.com/loomio/loomio/blob/master/client/shared/helpers/form.coffee

validation errors: https://github.com/loomio/loomio/blob/master/client/angular/components/validation_errors/validation_errors.coffee

**Comment by gdpelican** [comment]
Thanks for the detailed report. The fix you want is in `profile_page.haml`, by adding the `maxlength` and `md-maxlength` properties to the `%textarea` on line 33, like so:

```
%textarea.profile-page__short-bio-input{..., maxlength: "500", md-maxlength: "500"}
```

Which will add both a counter for the number of current characters typed, as well as stop you from continuing to type beyond 500 characters.

**Comment by robguthrie** [comment]
fixed in loomio 2.0
